code/game/objects/items/cards_ids.dm

```diff
@@ -233,91 +233,6 @@
 		windoor.obj_flags |= CMAGGED
 		return
 
-	if(istype(target, /obj/item/aiModule/core/full/crewsimov))
```

Comment:
Why are you deleting all this?

Reply:
Because its part of the CMAG code for Pranksimov

Reply:
(I also just pressed the revert button, I did nothing else)